py/desispec/workflow/os.py

```diff
@@ -0,0 +1,117 @@
+
```

Comment:
Let's pick a different name for this, to not confuse it with the standard library `os` module.  I think python3 has worked out non-ambiguous rules about if a parallel module does "import os" whether it means the system `os` or this `os.py` in the same directory, but let's not make ourselves have to remember that.  Maybe `desispec.workflow.util`?

Reply:
changed to utils